.github/workflows/jq.yml

```diff
@@ -0,0 +1,21 @@
+---
+# yamllint disable-line rule:truthy
+on:
+  push:
+    paths:
+      - '**.json'
+  pull_request:
+    paths:
+      - '**.json'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: docker://nanozoo/jq:1.6--0bafc15
```

Comment:
You don't need this. jq is already installed in GitHub hosted runner.

Reply:
I have tested you are right, I'll fix it. I am curious where you know this? Does github have any manual about it?

Reply:
OK I found it <https://docs.github.com/cn/actions/using-github-hosted-runners/about-github-hosted-runners#preinstalled-software>